### ISSUE TITLE
Matrix-types Description and Notes

### ISSUE THREAD

**Issue by jlahijani** [processwire/processwire-requests#430]
Right now, when adding a new matrix-type to a matrix field, you can input:

- field name
- field label
- fields
- item headers

Given the nature of the matrix field where each matrix-type has its own uniqueness to it, I think it would be helpful if a "Description" and "Notes" field could be added to matrix-types as well.  This could be used in providing editors vital information respective to the matrix-type right there.

**Comment by daun** [comment]
I agree this would be really helpful for in-context help messages. Currently we're using custom markup fieldtypes but that's rather brittle and unintuitive. An officially sanctioned way of doing this would be great.
